tsconfig.json

```diff
@@ -1,6 +1,6 @@
 {
   "compilerOptions": {
-    "lib": ["dom", "dom.iterable", "esnext"],
+    "lib": ["dom", "dom.iterable", "esnext","es2020"],
```

Comment:
why? 🤔  isn't this already included in "esnext"?

Reply:
I was getting the error "BigInt literals are not available when targeting lower than ES2020" when declaring BigInts before this. It was already in "esnext", I didn't really get what was going on, but it solved the error